timesketch/lib/analyzers/domain.py

```diff
@@ -110,10 +110,10 @@ def run(self):
                 cdn_counter[cdn_provider] += 1
 
             if domain in common_domains:
-                tags_to_add.append('common_domain')
+                tags_to_add.append('common-domain')
```

Comment:
can we just skip this tag?

I'm wondering if this is providing value or if it is just noise? we can add an attribute maybe, to make it easier to filter things out instead, WDYT?

Reply:
I would love to get less tags :) so SGTM. I'll move this to an attribute instead.